.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,3 +17,4 @@
 - [ ] Testing strategy adequately addresses listed risk(s).
 - [ ] Change is maintainable (easy to change, telemetry, documentation).
 - [ ] Release note makes sense to a user of the library.
+- [ ] Reviewer is aware of, and discussed the performance implications of this PR as reported in the benchmarks PR comment.
```

Comment:
Should we add guidelines on what's a reasonable increase in performance overhead? I know this is highly dependent on the change but we should try to establish a baseline and document overhead that's just unacceptable

Reply:
The benchmark platform automatically accounts for uncertainty in its measurements with confidence intervals, meaning it won't flag any scenarios as performance regressions/improvements unless it is significant enough. 

But I think it might be too difficult to quantify any baseline that is unacceptable since it can get quite subjective. The point of this checklist point is to remind both reviewer and author to have a discussion about performance implications of the change if there are any regressions/improvements period. IMO It's ultimately up to the author/reviewer to determine with is the acceptable amount of regression.